graphics/src/widget/canvas.rs

```diff
@@ -37,6 +37,8 @@ use iced_native::{
     Clipboard, Element, Length, Point, Rectangle, Shell, Size, Vector, Widget,
 };
 
+pub use crate::gradient::Gradient;
+
```

Comment:
Nit: Can be group with the `use::crate` import above

Reply:
Yeah just location wise. I know we don't have any consistency, but I try to strive for `StdExternalCrate` as seen here:

https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=import+gr#group_imports